composer.json

```
@@ -17,9 +17,9 @@
         }
     },
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=7.1.3",
```

We still can support php >=5.3.3 for now. 
Can you add a OR in the version please?
```suggestion
        "php": ">=5.3.3|>=7.1.3",
```